codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentBuilderGenerator.kt

```diff
@@ -150,8 +170,85 @@ class FluentBuilderGenerator(
                 }
                 """,
                 *scope,
+                "epilogue" to (
+                    handleEventStreamInitialResponse() ?: writable {
+                        rustTemplate("#{Operation}::orchestrate(&runtime_plugins, input).await", *scope)
+                    }
+                ),
+            )
+        }
+
+    private fun handleEventStreamInitialResponse(): Writable? {
+        if (codegenContext.protocolImpl?.httpBindingResolver?.handlesEventStreamInitialResponse(operation) != true) {
+            return null
+        }
+
+        // The variable eventStreamMember is guaranteed to be non-null because of the short-circuit above
+        val eventStreamMember =
```

Comment:
Is this guaranteed to return exactly one member?

Reply:
Yes, added explanatory comment in 872ab5d.